Graphics/GraphicsEngineD3D11/include/ShaderVariableManagerD3D11.hpp

```diff
@@ -138,7 +138,7 @@ class ShaderVariableManagerD3D11 : ShaderVariableManagerBase<EngineD3D11ImplTrai
             ShaderVariableD3D11Base<ConstBuffBindInfo, D3D11_RESOURCE_RANGE_CBV>{ParentLayout, ResIndex}
         {}
 
-        __forceinline void BindResource(const BindResourceInfo& BindInfo);
+        void BindResource(const BindResourceInfo& BindInfo);
```

Comment:
Why does `__forceinline` need to be removed here?
It should be defined as inline.
https://github.com/DiligentGraphics/DiligentCore/blob/master/CMakeLists.txt#L424

Reply:
For some reason it doesn't work in clang. Clang says it's not implemented if I leave it as is. I will double check if that define is going through.

Reply:
It is, but clang emits a warning if `inline` is used here regardless, which is treated as a compile error by `-Werror`:

```
In file included from D:/Repos/[game]/extlibs/DiligentCore/Graphics/GraphicsEngineD3D11/src/ShaderResourceBindingD3D11Impl.cpp:30:
In file included from D:/Repos/[game]/extlibs/DiligentCore/Graphics/GraphicsEngineD3D11/include\ShaderResourceBindingD3D11Impl.hpp:38:
D:/Repos/[game]/extlibs/DiligentCore/Graphics/GraphicsEngineD3D11/include\ShaderVariableManagerD3D11.hpp:141:21: error: inline function 'Diligent::ShaderVariableManagerD3D11::ConstBuffBindInfo::BindResource' is not defined [-Werror,-Wundefined-inline]
  141 |         inline void BindResource(const BindResourceInfo& BindInfo);
      |                     ^
D:/Repos/[game]/extlibs/DiligentCore/Graphics/GraphicsEngine/include\ShaderResourceVariableBase.hpp:671:43: note: used here
  671 |         static_cast<ThisImplType*>(this)->BindResource(BindResourceInfo{pObject, Flags});
      |                                           ^
```

Reply:
OK, looks like `__forceinline` indeed needs to be removed here.